starter/source/materials/mat/mat062/hm_read_mat62.F

```diff
@@ -333,6 +325,13 @@ SUBROUTINE HM_READ_MAT62(
      & 5X,'VISCOUS STRESS FORMULATION . . . . . . =',I8 /
      &10X,' 0 : VISCOUS STRESS IS DEVIATORIC ' , /
      &10X,' 1 : VISCOUS STRESS IS SPHERICAL AND DEVIATORIC' )
+ 1010 FORMAT(/
+     & 5X,'LONG-TERM RIGIDITY FLAG . . . . . =',I8 /
+     & 5X,'INITIAL ELASTIC MODULUS IS THE INTANTANEOUS RIGIDITY ' , / )
+ 1020 FORMAT(/
+     & 5X,'LONG-TERM RIGIDITY FLAG . . . . . =',I8 /
+     & 5X,' INITIAL ELASTIC MODULUS IS THE LONG-THERM RIGIDITY. ',/
```

Comment:
here it is LONG TERM , not LONG-THERM

Reply:
May I also notice this typo : **INST**ANTANEOUS  instead of **INT**ANTANEOUS